src/coreclr/jit/codegenarmarch.cpp

```diff
@@ -1169,14 +1170,19 @@ void CodeGen::genPutArgSplit(GenTreePutArgSplit* treeNode)
 
             if (regIndex >= treeNode->gtNumRegs)
             {
+                if (firstOnStackOffs == UINT_MAX)
+                    firstOnStackOffs = use.GetOffset();
+
                 var_types type = nextArgNode->TypeGet();
                 emitAttr  attr = emitTypeSize(type);
 
+                unsigned offset = treeNode->getArgOffset() + use.GetOffset() - firstOnStackOffs;
+                // We can't write beyound the outgoing area area
```

Comment:
beyound -> beyond
area area -> ?

Reply:
Thanks, fixed this comment in a few more places in the file